src/main.ts

```diff
@@ -27,7 +27,15 @@ async function run(): Promise<void> {
     }
 
     stateHelper.setCleanup(inputs.cleanup);
-    await exec.exec(`${daggerBin} ${inputs.args} --log-format plain`);
+
+    if (inputs.args) {
+      inputs.cmds.unshift(inputs.args);
+      core.warning(`'args' is being deprecated, please use 'cmds'`);
```

Comment:
Not needed with `deprecationMessage` in `action.yml`. See https://github.com/dagger/dagger-for-github/pull/47/files#r848922192